_posts/2024-12-19-cosmicqc.md

```diff
@@ -0,0 +1,313 @@
+This project is the first of hopefully many examples where we can show the importance of single-cell quality control!
+
+---
+
+## What's next?
+
+We have already implemented this multiple other projects!
+Now, I am not going to sit here writing this blog and pretend like coSMicQC works without any fault.
+I want to be very clear, this method is very good when applying it to cells without perturbation (e.g., happy, normal cells).
+We applying coSMicQC to a dataset where there are cells treated with many different perturbations, we saw that nuclei with very interesting phenotypes ended up failing quality control (red) at a higher rate than those that looked more normal.
+
+{%
+  include figure.html
+  image="images/blog/cosmicqc/phenotype_example.png"
+  width="100%"
+%}
+> Example of passing (green) and failing (red) single cells for a normal phenotype FOV (left) and interesting phenotype FOV (right).
+
+This means that unfortunately, coSMicQC is not robust to some phenotypes and might over-correct the data.
+But, quality control is not a perfect science.
+The purpose of quality control, regardless of field, is to maximize the amount of poor quality cell removed while minimizing the amount of good quality cells being wrongly eliminated from the dataset.
+This means that no matter what, you must expect some false positives, but we can attempt to control for this.
+
+Given that we know the current limitations of the current simple methodology of coSMicQC, we are looking to improve by finding new methods to implement into our software.
+Some of the current ideas we are thinking of testing include DBSCAN (proposed by Erik Serrano aka @axiomcura) and PCA or UMAP to utilize either all features or selected features to find outliers based on clusters.
+As well, given the results from that paper I referenced previously, we might considering incorporating the HBOS method from the PyOD package into coSMicQC.
+
+Our big goal for 2025 is publish a paper for coSMicQC, which means we will be testing on more datasets.
+We plan on updating this blog with more sections that include the results from evaluating on more data, so please look out for more!
+
+---
+
+## Final thoughts
+
+If you have gotten to the end of this blog and have an idea that you want to propose, please feel free to [add an issue](https://github.com/WayScience/coSMicQC/issues) to the coSMicQC GitHub!
+We would really like to make a push towards standardizing this process within the traditional image-based profiling workflow.
+We hope to build a community around single-cell quality control for morphology profiles, so we can lead the field to a future with higher-quality datasets.
+
+Thank you for reading, and happy profiling! 👩🏻‍💻
+
+### Acknowledgements
+
+Thank you to the Way Lab for their constant support!
+Thank you especially to Dave Bunten for their direct contributions to coSMicQC and to striving for improved data quality and standardization!
```

Comment:
might want to also add Erik

Reply:
I will add a mention directly to Erik for his continued interest and ideas! :)